README.md

```diff
@@ -74,6 +74,8 @@ These functions enhance security and support localization efforts in WordPress b
 
 - **Patterns with images**
 
+You can find the images used in the theme designs, in the following link: https://tinyurl.com/tt5-images
```

Comment:
Suggestion:

`You can find the images used in the theme designs, [in this folder](https://tinyurl.com/tt5-images).`